tagger.go

```diff
@@ -134,6 +157,29 @@ func updateAllInstanceTags(ctx context.Context, client linodego.Client, tagMap i
 	return nil
 }
 
+func genReport(report reportMap) error {
+	// github.com/jedib0t/go-pretty/v6/table
+	t := table.NewWriter()
+	t.SetOutputMirror(os.Stdout)
+	t.AppendHeader(table.Row{"tag", "Linodes Changed", "Linodes", "Added/Removed"})
+	for tag := range report {
+		if len(report[tag]["absent"]) > 0 {
+			// truncate linode list lag arbitrarily as it can get too long
+			// column maxwidth and row restrictions don't work well for this
+			report[tag]["absent_linodes"] = report[tag]["absent_linodes"][:45] + "..."
+			t.AppendRow(table.Row{tag, report[tag]["absent"], report[tag]["absent_linodes"], "Removed"})
```

Comment:
This feels like an unnecessary re-assignment in the map to truncate the results.
Can we change this to have the call to `AppendRow()` pass the slice index range instead of re-assigning it in the map? 

```suggestion
                         // needs `go fmt`'ing
			t.AppendRow(table.Row{
			    tag, 
			    report[tag]["absent"], 
			    report[tag]["absent_linodes"][:45] + "...", 
			    "Removed",
			})
```